static/js/tools/stat_var/util.ts

```diff
@@ -34,6 +34,21 @@ export function getUrlToken(param: string): string {
   return urlParams.get(param);
 }
 
+/**
+ * Returns token for URL param, or the given default.
+ * @param param URL param
+ * @def def default value
+ * @returns Corresponding token
+ */
+export function getUrlTokenOrDefault(param: string, def: string): string {
+  const urlParams = new URLSearchParams(window.location.hash.split("#")[1]);
+  const res = urlParams.get(param);
```

Comment:
nit: could just do `res = getUrlToken(param)` instead of re-writing the code?

Reply:
yes!